lockable_resource/migrations/0008_lockableresource_locked_time.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2 on 2022-06-20 12:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("lockable_resource", "0007_lockableresource_associated_queue"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="lockableresource",
+            name="locked_time",
```

Comment:
Instead of `locked_time` , can we term it as lease period ? in that way we can introduce lease semantics like lease_period, lease_expiry etc. WDYT ?

Reply:
Sounds good using the term period in this case. So maybe locked_period is a better usable field. 
As @petr-balogh mentioned, using logic like the expiry of a locked resource is something that is better to have as a separated component of the entire application because it only covers our specific use-case. The intention is to deliver it to more teams.
When we will start writing a separate component to cover this, I think we can use field names such as `lease_period`, `lease_expiry`, etc.

Reply:
Addressed in #25 
I added a locked period property.
Correction for the previous comment: The field you suggested to edit is the **exact** time object that shows when the resource is locked. 
But for calculations of time difference, I use the term `locked_period`